gui/src/components/health/cortx-health-graphical.vue

```diff
@@ -0,0 +1,544 @@
+        .attr("dy", "45px")
+        .attr("x", function(d: any) {
+          if (
+            d.data.children &&
+            d.data.children.length &&
+            d.children &&
+            d.children.length
+          ) {
+            return "152.5";
+          } else {
+            return "150";
+          }
+        })
+        .text(function(d: any) {
+          if (d.children) {
+            return "-";
+          } else if (!d.children && d.data.children && d.data.children.length) {
+            return "+";
+          } else {
+            return " ";
+          }
+        })
+        .style("cursor", "pointer")
+        .on("click", click);
+      // UPDATE
+      let nodeUpdate = nodeEnter.merge(node);
+      // Transition to the proper position for the node
+      nodeUpdate
+        .transition()
+        .duration(duration)
+        .attr("transform", function(d: any) {
+          return "translate(" + d.y + "," + d.x + ")";
+        });
+      // Remove any exiting nodes
+      let nodeExit = node
+        .exit()
+        .transition()
+        .duration(duration)
+        .attr("transform", function(d: any) {
+          return "translate(" + source.y + "," + source.x + ")";
+        })
+        .remove();
+      // On exit reduce the node circles size to 0
+      nodeExit.select("rect")
+        .attr("width", "140")
+        .attr("height", "100")
+        .attr("stroke-width", 1);
+
+      nodeExit.select("text");
+      // ****************** links section ***************************
+      // Update the links...
+      let link = svg.selectAll("path.link").data(links, function(d: any) {
+        return d.id;
+      });
+      // Enter any new links at the parent's previous position.
+      let linkEnter = link
+        .enter()
+        .insert("path", "g")
+        .attr("class", "link")
+        .attr("d", function(d: any) {
+          let o = { x: source.x0, y: source.y0 };
+          return diagonal(o, o);
+        })
+        .style("fill", "none")
+        .style("stroke", "#0832A0")
+        .style("stroke-width", "1px")
+        .style("stroke-dasharray", "2 2");
+      // UPDATE
+      let linkUpdate = linkEnter.merge(link);
+      // Transition back to the parent element position
+      linkUpdate
+        .transition()
+        .duration(duration)
+        .attr("d", function(d: any) {
+          return diagonal(d, d.parent);
+        });
+      // Remove any exiting links
+      let linkExit = link
+        .exit()
+        .transition()
+        .duration(duration)
+        .attr("d", function(d: any) {
+          let o = { x: source.x, y: source.y };
+          return diagonal(o, o);
+        })
+        .remove();
+
+      // Store the old positions for transition.
+      nodes.forEach(function(d: any) {
+        d.x0 = d.x;
+        d.y0 = d.y;
+      });
+      // Creates a curved (diagonal) path from parent to the child nodes
+      function diagonal(s: any, d: any) {
+        let path;
```

Comment:
this should be const as it is not being reassigned. also you can simply return template literal

Reply:
Variable updated to const.